README.md

```diff
@@ -151,6 +153,12 @@ The following code block shows the structure of the configuration file and provi
 
         // The number of threads dedicated to servicing client requests.
         "threads": 10,
+
+        // The size of the buffer when calling PutObject.
+        "put_object_buffer_size_in_bytes": 8192,
+
+        // The size of the buffer when calling GetObject 
```

Comment:
Missing a period at the end of the sentence.

Reply:
Well, it is technically not a proper sentence but I will add the period.

Reply:
I added the period.  I always struggle when I am writing a sentence fragment but agree consistency is important.